contrib/blueflood-docker/docker-compose.yml

```diff
@@ -1,5 +1,5 @@
 blueflood:
-  build: .
+  image: goru97/blueflood
```

Comment:
I can host the image on rackerlabs. @stackedsax do you have credentials for this account https://hub.docker.com/u/rackerlabs/ ?